ZenPacks/zenoss/LinuxMonitor/modeler/plugins/zenoss/cmd/linux/os_service.py

```diff
@@ -319,20 +319,21 @@ def check_services_modeled(model_list, ignore_list, title):
         if re.match(name, title):
             return False
 
-    for name in model_list:
-        if re.match(name, title):
-            return True
-    return False
+    if model_list:
+        for name in model_list:
+            if re.match(name, title):
+                return True
+        return False
+    else:
+        return None
```

Comment:
I've edited this part where we _only_ model service names that match in zLinuxModelServices. If it's not in the list, we do not model.